packages/server/lib/crons/usage.ts

```diff
@@ -0,0 +1,88 @@
+import tracer from 'dd-trace';
+import * as cron from 'node-cron';
+
+import { records } from '@nangohq/records';
+import { connectionService } from '@nangohq/shared';
+import { getLogger, metrics, report, flagHasUsage } from '@nangohq/utils';
+
+import { envs } from '../env.js';
+
+const logger = getLogger('cron.exportUsage');
+const cronMinutes = envs.CRON_EXPORT_USAGE_MINUTES;
+
+export function exportUsageCron(): void {
+    if (!flagHasUsage || cronMinutes <= 0) {
```

Comment:
added a flag to disable usage reporting by default (so nothing happens on Enterprise setup).